app/styles/templates/orders/_index.scss

```diff
@@ -112,7 +110,9 @@ $header-height: 3em;
     }
   }
 
-
+  .infinite-list-container {
+    overflow-y: auto !important;
```

Comment:
I will suggest to check CSS on infinity-scroll plugin and decide it this really needed? As infinity-scroll is our own plugin.

Reply:
Okay got it